### PR TITLE
docs: using kustomize for adjusting resources

### PR DESCRIPTION
**Pull request by omerap12**
Fixes: #481

### CODE REVIEW

File: site/content/en/docs/installation/_index.md [hunk 1]
```diff
@@ -65,6 +65,82 @@ kubectl apply -f https://github.com/kubernetes-sigs/jobset/releases/download/$VE
...
+patches:
+  - path: resource_patch.yaml
+```
+
+### Step 5: Include the Patch in Your Kustomization
+
+Apply the configuration to your Kubernetes cluster using Kustomize and kubectl:
+
+```shell
+kustomize build . | kubectl apply -f -
```
Comment (danielvegamyhre) [inline site/content/en/docs/installation/_index.md:142]: Can we just use `kubectl apply -k ...` here? Since kustomize is included in kubectl now

Comment (omerap12) [reply site/content/en/docs/installation/_index.md:142]: Sure

**Review by danielvegamyhre — COMMENTED** [review]
Looks good, did you manually test these steps and verify they worked?

File: site/content/en/docs/installation/_index.md [hunk 2]
```diff
@@ -65,6 +65,82 @@ kubectl apply -f https://github.com/kubernetes-sigs/jobset/releases/download/$VE
...
+    spec:
+      containers:
+      - name: manager
+        resources:
+          requests:
+            cpu: "1"      # Updated CPU request
+            memory: "256Mi"  # Updated memory request
+          limits:
+            cpu: "4"      # Updated CPU limit
+            memory: "1Gi"  # Updated memory limit
```
Comment (googs1025) [inline site/content/en/docs/installation/_index.md:122]: just a nit, Let's keep the comment tidy.

Comment (omerap12) [reply site/content/en/docs/installation/_index.md]: Done

**Comment by omerap12** [comment]
> Looks good, did you manually test these steps and verify they worked?

Yes

**Comment by danielvegamyhre** [comment]
/lgtm
/approve

Thanks @omerap12!

**Pull request merged**